java/flyway/Portfile

```diff
@@ -27,7 +28,8 @@ checksums           rmd160  d30549f7cbe20ecd2181945b9b8916c5592abc9e \
                     sha256  ff91611330f6f993a3c02f4102ea8e136b39248aed47b4da91936680b975c984 \
                     size    49882528
 
-depends_run         bin:java:kaffe
+java.version        1.8+
+java.fallback       openjdk11
```

Comment:
The version information is based on what is posted at https://flywaydb.org/download/: "Java 8/9/10/11/12/13 compatibility". Since it is compatible with the latest LTS Java version (Java 11), I have specified that as the fallback version (what MacPorts will install if no other compatible Java installation is already present).